db/migrate/20220705210803_create_likes.rb

```diff
@@ -0,0 +1,9 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :users, null: false, foreign_key: true
```

Comment:
- [ ] Let's reference author, this way we can achieve the `author_id` shown on the ERD diagram.

Reply:
@elisha2kyakpo1 Hello Mr.
How can I  reference `author_id` with this command `rails generate migration Post user:references` ?

Reply:
@fed1k, try explicitly adding this line of code `t.references :author, null: false, foreign_key: {to_table: :users}`, but again like I said, you should make some changes at model level associations as well.

Reply:
Thanks for the help @elisha2kyakpo1